test/test_nn.py

```diff
@@ -27,7 +27,8 @@
 from torch.nn.parallel._functions import Broadcast
 from common_nn import NNTestCase, ModuleTest, CriterionTest, TestBase, \
     module_tests, criterion_tests, TEST_CUDA, TEST_MULTIGPU, TEST_CUDNN, \
-    TEST_CUDNN_VERSION, nllloss_reference, nllloss2d_reference
+    TEST_CUDNN_VERSION, nllloss_reference, nllloss2d_reference, \
+    smoothl1loss_reference
```

Comment:
it might be nicer to have a dict of reference functions that you import here, and then index by module name. We wouldn't need to modify the import list every time then